vendor/modules.txt

```diff
@@ -200,8 +200,8 @@ github.com/k8stopologyawareschedwg/deployer/pkg/objectupdate/rte
 github.com/k8stopologyawareschedwg/deployer/pkg/objectupdate/sched
 github.com/k8stopologyawareschedwg/deployer/pkg/options
 github.com/k8stopologyawareschedwg/deployer/pkg/validator
-# github.com/k8stopologyawareschedwg/noderesourcetopology-api v0.1.1
-## explicit; go 1.16
+# github.com/k8stopologyawareschedwg/noderesourcetopology-api v0.1.2
+## explicit; go 1.20
```

Comment:
In general, shouldn't we move forward to 1.22 already?

Reply:
this is auto-generated by the golang tooling and AFAIK refers to the module's go version: https://github.com/k8stopologyawareschedwg/noderesourcetopology-api/blob/master/go.mod#L3